AGENTS.md

```
@@ -1,8 +1,50 @@
+- `cli` – binary launcher
+- `config` – configuration library
+- `kernel` – core functionality and database layer
+
+Read the [README.md](./README.md) for an introduction and
+[CONTRIBUTING.md](./CONTRIBUTING.md) for contributor guidelines.
+
+## Development setup
+
+1. Install `sqlx-cli`.
+2. Add a `.env` file at the repository root containing:
+   ```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified
null

(MD040, fenced-code-language)